### PR TITLE
Move custom colors used by Khan into macros.js

### PR DESCRIPTION
**Pull request by danielhollas**
This is a step towards resolving #28. I moved the color definitions from `Options.js` and `functions/color.js` into `macro.js`.

### CODE REVIEW

**Comment by danielhollas** [comment]
I wonder whether the color mapping should not be in a separate JSON file. There are other places in the Khan Academy codebase where these are used so this might be valuable...

**Comment by kevinbarabash** [comment]
@danielhollas thanks for the PR.  This saves a little space so I think we should go ahead with this PR.  

Long term though I think we should move these custom colors out of the KaTeX codebase altogether.  The downside of doing so right now is that we'd have to reparse these colors each time we render any math on khanacademy.org on in the mobile apps.  I think we should implement `katex.globalEval` as suggested in #750 before moving these custom colors out of KaTeX.

File: src/Options.js [hunk 1]
```diff
@@ -299,81 +299,13 @@ class Options {
...
+     * Gets the CSS color of the current options object
      */
     getColor(): string | void {
         if (this.phantom) {
             return "transparent";
-        } else if (
-            this.color != null &&
-            Options.colorMap.hasOwnProperty(this.color)
-        ) {
-            return Options.colorMap[this.color];
```
Comment (kevinbarabash) [inline src/Options.js]: Nice

File: src/functions/color.js [hunk 2]
```diff
@@ -55,42 +55,6 @@ defineFunction({
     mathmlBuilder,
 });
 
-// TODO(kevinb): define these using macros
-defineFunction({
```
Comment (kevinbarabash) [inline src/functions/color.js]: yay

**Pull request merged**